tools/docker/docker-compose.ts-integration.yaml

```diff
@@ -0,0 +1,74 @@
+version: '3.5'
+
+services:
+  integration-ts:
+    container_name: ts-integration-tests
+    image: smartcontract/ts-integration
+    build:
+      context: ../../
+      dockerfile: tools/docker/ts-integration.Dockerfile
+      args:
+        - SRCROOT
+    volumes:
+      # Forwards the local Docker socket to the container.
+      - /var/run/docker.sock:/var/run/docker.sock
+    depends_on:
+      - node
+      - echo-server
+      - external-adapter
+    environment:
+      - SRCROOT
+      - GETH_MODE
+      - CHAINLINK_DEV=TRUE # need this for service agreements subcommand
+      - CHAINLINK_URL
+      - ETH_HTTP_URL
+      - ECHO_SERVER_URL
+      - EXTERNAL_ADAPTER_URL
+      - CLIENT_NODE_URL # used to send remote commands
+      - CYPRESS_JOB_SERVER_PORT
+      - CYPRESS_JOB_SERVER_HOST
+      - CYPRESS_EXPLORER_URL
+      - CYPRESS_CHAINLINK_URL=$CHAINLINK_URL
+    secrets:
+      - apicredentials
+
+  echo-server:
+    container_name: echo-server
+    image: smartcontract/echo-server
+    build:
+      context: ../../
+      dockerfile: tools/echo-server/Dockerfile
+      args:
+        - SRCROOT
+    environment:
+      - PORT=$ECHO_SERVER_PORT
+
+  cypress-job-server:
```

Comment:
Is cypress needed for these tests?

Reply:
They aren't (yet) but they will be once we convert the other tests in the `ethereum_test`

Reply:
I guess I'll remove for now and whoever converts the other tests can add them back